### PR TITLE
Scala 2.11 support

### PR DESCRIPTION
**Pull request by patrickdsouza**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -69,4 +77,3 @@ publishTo := {
   else
     Some("releases"  at nexus + "service/local/staging/deploy/maven2")
 }
-
```
Comment (dspeiser) [inline build.sbt:72]: No newline at eof?

File: build.sbt [hunk 2]
```diff
@@ -19,22 +19,30 @@ sonatypeSettings
...
-version := "0.2"
+version := "0.3"
 
 description := "A Scala client for the Presto SQL engine"
 
 homepage := Some(url("https://github.com/nezihyigitbasi/presto-scala-client"))
 
+resolvers ++= Seq(Resolver.sonatypeRepo("releases"),
+    Resolver.sonatypeRepo("snapshots"),
+    Resolver.bintrayRepo("scalaz", "releases"),
```
Comment (dspeiser) [inline build.sbt:34]: Is scalaz needed here?

Comment (patrickdsouza) [reply build.sbt:34]: Well that's the recommendation for its usage over here https://github.com/megamsys/newman/. https://github.com/stackmob/newman is no longer maintaining the fork and the most active fork is available at https://github.com/megamsys/newman/

File: build.sbt [hunk 3]
```diff
@@ -19,22 +19,30 @@ sonatypeSettings
...
-  "com.facebook.presto" % "presto-client" % "0.114",
-  "com.typesafe.scala-logging" % "scala-logging-slf4j_2.10" % "2.1.2",
+  "io.megam" %% "newman" % "1.3.12",
+  "com.facebook.presto" % "presto-client" % "0.157",
+  "com.typesafe.scala-logging" % "scala-logging-slf4j_2.11" % "2.1.2",
   "org.slf4j" % "slf4j-api" % "1.7.1",
   "org.slf4j" % "log4j-over-slf4j" % "1.7.1",  // for any java classes looking for this
   "ch.qos.logback" % "logback-classic" % "1.0.3",
-  "org.scalatest" % "scalatest_2.10" % "2.2.2" % "test"
+  "org.scalatest" % "scalatest_2.11" % "2.2.4" % "test"
```
Comment (Seanstoppable) [inline build.sbt]: Just change this to
```
"org.scalatest" %% "scalatest" % "2.2.4" % "test"

**Review by Seanstoppable — APPROVED** [review]

**Pull request merged**